RULES.md

```diff
@@ -0,0 +1,72 @@
+This project validates feeds up to version 2.2 of the [JSON Schemas](https://github.com/NABSA/gbfs/blob/master/gbfs.md).
+# Files presence
+The validator will display a message for all files that are present. A message will be displayed for each missing file. See examples below.
+
+Missing file but not required:
+![missing_not_required](https://user-images.githubusercontent.com/63653518/133173202-459fc2e3-6c4b-4748-8a37-94ed362210c0.png)
+Missing file and required:
```

Comment:
Check the markdown rendering. Maybe add an empty line.

Reply:
Done and I reseized the images